test/test_ops.py

```diff
@@ -1815,6 +1815,8 @@ def test_refs_are_in_decomp_table(self, op):
     "nn.functional.huber_loss",
     "nn.functional.logsigmoid",
     "nn.functional.multilabel_soft_margin_loss",
+    "unfold",  # fails an aliasing assert
```

Comment:
In the future, could we fix these asserts before landing ? Or tag me on them ? cc @bdhirsh

Reply:
There is also a `disable_meta` flag you could have applied that wouldn't introduce failures here

Reply:
Sure, will do in the future.
It was not clear to me what the assert was, whether it was important or how to fix it. Adding more descriptive error messages would help.